sse2neon.h

```diff
@@ -70,6 +72,8 @@
 #ifndef ALIGN_STRUCT
 #define ALIGN_STRUCT(x) __declspec(align(x))
 #endif
+#define likely(x) (x)
```

Comment:
Add guard like this:
```c
#ifndef likely
#define likely(x) (x)
#endif
```
For compiler lacking of `push_macro` feature, it is necessary to avoid the pollution of macros.